modules/gui/src/com/haulmont/cuba/gui/export/ExcelExporter.java

```diff
@@ -632,7 +632,7 @@ protected void createDataGridRow(DataGrid dataGrid, List<DataGrid.Column> column
 
         int level = 0;
         if (dataGrid instanceof TreeDataGrid) {
-            level = ((TreeDataGrid) dataGrid).getLevel(itemId);
+            level = ((TreeDataGrid) dataGrid).getLevel((Entity) instance);
```

Comment:
Change the type of the `instance` variable to `Entity` and rename it. (see line :631).